### DISCUSSION TITLE
Validation Accuracy Discrepancy in Custom Dataset Training

### DISCUSSION THREAD

**Discussion by mano066** [open-mmlab/mmclassification#1805] [Q&A]
I am facing an issue after training a model using a custom dataset. The problem I'm encountering is that the validation accuracy and the confusion matrix results do not match up as expected.

**Comment by sinyoung-park** [comment]
that is not correct. the confusion matrix is created by test accuracy.
how to make a  Json file? Generaly Json is maked test result.
